bottle_server.py

```diff
@@ -58,6 +60,19 @@ def route_static(filepath=None):
         return template('index', root=appPath, get_url=app.get_url)
     return static_file(filepath, root=join(appPath, 'static'))
 
+@get('/version', name='version')
+def get_version():
+    version_backend=subprocess.Popen(['./shell-scripts/run-version-checker.sh', CF], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
Instead of the new `CF` variable, shouldn't this use the `cfPath`?

Reply:
It actually doesn't work on my local machine when I do cfPath, but I changed it to cfPath as I assume that is something unique for my machine